### ISSUE TITLE
Cannot read property 'type' of undefined

### ISSUE THREAD

**Issue by vilmysj** [danialfarid/ng-file-upload#1370]
I am getting this error when html element such as `a` is dragged over upload area.
This could be common error in page where all content is upload area.

Same error can be reproduced in [DEMO](https://angular-file-upload.appspot.com/) page.

![ng-upload-error](https://cloud.githubusercontent.com/assets/4406221/13177501/3028e37e-d722-11e5-8a48-e60f96362f23.jpg)

error stack trace:
```js
TypeError: Cannot read property 'type' of undefined
    at ng-file-upload.js:1363
    at Object.forEach (angular.js:325)
    at validateAsync (ng-file-upload.js:1360)
    at upload.validate (ng-file-upload.js:1397)
    at upload.updateModel (ng-file-upload.js:642)
    at updateModel (ng-file-upload.js:1890)
    at ng-file-upload.js:1847
    at wrappedCallback (angular.js:11572)
    at angular.js:11658
    at Scope.$eval (angular.js:12701)
```

**Comment by danialfarid** [comment]
https://github.com/danialfarid/ng-file-upload/issues/1325#issuecomment-187240142

**Comment by danialfarid** [comment]
Fixed at 12.0.3
